code/modules/mob/dead/observer/observer.dm

```diff
@@ -735,4 +735,18 @@ This is the proc mobs get to turn into a ghost. Forked from ghostize due to comp
 			LAZYINITLIST(mob_eye.observers)
 			mob_eye.observers |= src
 			mob_eye.hud_used.show_hud(1,src)
-			observetarget = mob_eye
\ No newline at end of file
+			observetarget = mob_eye
+
+/mob/dead/observer/verb/register_pai_candidate()
+	set category = "Ghost"
+	set name = "pAI Setup"
+	set desc = "Upload a fragment of your personality to the global pAI databanks"
+
+	register_pai()
+
+/mob/dead/observer/proc/register_pai()
```

Comment:
pass user in as an argument to this proc, we prefer to keep use of usr confined to the highest level procs where possible (topic and verbs)